File: Daily Test/C/Smaller Unit Digits 15.4.2026.c
#include <stdio.h>

int main() {
    int a, b;
    scanf("%d %d", &a, &b);

    int num;

    if(a % 10 == b % 10) {
        num = (a > b) ? a : b;
    } else {
        if(a % 10 < b % 10)
            num = a;
        else
            num = b;
    }

    printf("%d", num);
    return 0;
}

File: Daily Test/Java/Smaller Unit Digits 15.4.2026.java
import java.util.*;

public class Hello {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);

        int a = s.nextInt();
        int b = s.nextInt();

        int num;

        if(a % 10 == b % 10) {
            num = Math.max(a, b);
        } else {
            if(a % 10 < b % 10)
                num = a;
            else
                num = b;
        }

        System.out.println(num);
    }
}